src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -4,13 +4,16 @@ import { NavigationMixin } from 'lightning/navigation';
 import messageLoading from '@salesforce/label/c.labelMessageLoading';
 import geSave from '@salesforce/label/c.labelGeSave';
 import geCancel from '@salesforce/label/c.labelGeCancel';
-import { showToast, handleError } from 'c/utilTemplateBuilder';
+import { showToast, handleError, getQueryParameters, getRecordFieldNames, setRecordValuesOnTemplate } from 'c/utilTemplateBuilder';
 import { getRecord } from 'lightning/uiRecordApi';
 import FORM_TEMPLATE_FIELD from '@salesforce/schema/DataImportBatch__c.Form_Template__c';
 import TEMPLATE_JSON_FIELD from '@salesforce/schema/Form_Template__c.Template_JSON__c';
 
 export default class GeFormRenderer extends NavigationMixin(LightningElement) {
-    @api sections = [];
+    @api recordId = '';
+    @api record;
```

Comment:
Is this the recordId and record that matches the account/contact being passed in? I feel like that's a bit misleading in this context, where I might expect the recordId to match either the DI or the Opportunity, and if we CAN be flexible on the naming, something like "relatedRecordId" or "donorRecordId" or "sourceRecordId" might be more descriptive.